bug: Patching of model costs for past traces does not work

### Describe the bug

I am running the exact same commands as mentioned in https://langfuse.com/docs/deployment/self-host#migrate-models to patch my past traces with the latest model costs.

However I am getting this error:
Error: Cannot find module '/home/[user]/langfuse/web/node_modules/@langfuse/shared/dist/src/db.js'
code: 'MODULE_NOT_FOUND',
path: '/home/[user]/langfuse/web/node_modules/@langfuse/shared/package.json'

Expected: DB migration should take place seamlessly

### To reproduce

Run the exact same commands as mentioned in https://langfuse.com/docs/deployment/self-host#migrate-models

### SDK and container versions

Running latest version of langfuse

### Additional information

_No response_

### Are you interested to contribute a fix for this bug?

No

--- a/turbo.json
+++ b/turbo.json
@@ -17,7 +17,7 @@
     "db:push": {},
     "db:seed": {},
     "models:migrate": {
-      "dependsOn": ["db:generate"]
+      "dependsOn": ["db:generate", "@langfuse/shared#build"]
     },
     "db:seed:examples": {},
     "dev": {

